### PR TITLE
Adjust tests in Indexing folder for string option

### PR DESCRIPTION
**Pull request by phofl**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

**Comment by phofl** [comment]
This should be ready for a look

File: pandas/tests/indexing/test_coercion.py [hunk 1]
```diff
@@ -824,6 +826,7 @@ def replacer(self, how, from_key, to_key):
             raise ValueError
         return replacer
 
+    @pytest.mark.skipif(using_pyarrow_string_dtype(), reason="TODO: test is to complex")
```
Comment (mroeschke) [inline pandas/tests/indexing/test_coercion.py:829]: Was the failure here do due a bug or something expected to work? Would be good to mention that here

Comment (phofl) [reply pandas/tests/indexing/test_coercion.py:830]: I think everything works as expected, but the parametrisation is so complex that I couldn't figure this out to adjust efficiently, that's why I am skipping for now, added a comment

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @phofl

**Pull request merged**